src/status/libstatus/contacts.nim

```diff
@@ -26,13 +24,11 @@ proc getContacts*(): JsonNode =
     return %* []
   return response["result"]
 
-proc saveContact*(id: string, ensVerified: bool, ensName: string, ensVerifiedAt: int, ensVerificationRetries: int, alias: string, identicon: string, systemTags: seq[string], localNickname: string): string =
+proc saveContact*(id: string, ensVerified: bool, ensName: string, alias: string, identicon: string, systemTags: seq[string], localNickname: string): string =
   let payload = %* [{
       "id": id,
       "name": ensName,
       "ensVerified": ensVerified,
-      "ensVerifiedAt": ensVerifiedAt,
```

Comment:
These arent necessary to create a contact?

Reply:
It got removed in status-go and also we never used them